packages/calypso-build/webpack/util.js

```diff
@@ -14,4 +19,52 @@ function cssNameFromFilename( name ) {
 	}
 }
 
-module.exports = { cssNameFromFilename };
+function createIncrementalProgressPlugin() {
```

Comment:
It probably doesn't matter that much, but the webpack convention for plugins is exporting class constructors and instantiate them with `new`:
```js
new IncrementalProgressPlugin()
```

Reply:
yeah, but that led to this bit of fun code: 

```js
function IncrementalProgressPlugin() {
  // fake out the ctor and return a different object instance
  // this works but feels weirdly gross to me.
  return new WebpackProgressPlugin( createProgressHandler() );
}
```

Reply:
done!